rfcs/protocols/asset-transfer/generic.md

```diff
@@ -40,4 +58,4 @@ _TBD: list and discuss the library functions that support transfers, excluding s
 
 ## Safety and Liveness
 
-_TBD_
+At no point in time, asset *M* is available in both the ledgers *A* and *B*. This is ensured by allowing minting of asset by Alice in ledger *A* only if the asset was not already minted by Bob in ledger *B*. Also note that Alice could mint the asset *M* in ledger *A* anytime after the period *t*.
```

Comment:
"At no point in time is asset *M* available in both......."

Reply:
Done